core/ir/opnd.h

```diff
@@ -1443,6 +1444,7 @@ enum {
     NULL_kind,
     IMMED_INTEGER_kind,
     IMMED_FLOAT_kind,
+    IMMED_DOUBLE_kind,
```

Comment:
Inserting in the middle is a binary compatibility break in this exposed enum.  This would have to be documented as such and the version numbers jumped including the oldest compatible.  Better to insert at the end.

Reply:
Done.